### PR TITLE
 Fixed typing in contentEditable for new Chrome (closes #1416)

### LINKED ISSUE #1416: qunit tests for contentEditable fail in Chrome 58 beta
### Are you requesting a feature or reporting a bug?

bug
https://www.screencast.com/t/P0p59mIPmx

### What is the current behavior?

contentEditable tests fail

### What is the expected behavior?

qunit tests should pass

### How would you reproduce the current behavior (if this is a bug)?

run qunit tests with chrome 58 bets

### Specify your

* operating system: win 10 x64
* testcafe version: 0.14.0
* node.js version: 5.4.1

### PR DESCRIPTION
**Pull request by georgiy-abbasov**

### CODE REVIEW

**Comment by georgiy-abbasov** [comment]
Chrome 58 already released, but it's not present in [SL platform configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator) yet, only beta is available.
\cc @AlexanderMoskovkin, @helen-dikareva

File: src/client/automation/playback/type/type-text.js [hunk 1]
```diff
@@ -31,7 +31,9 @@ function _updateSelectionAfterDeletionContent (element, selection) {
     var startNode      = selection.startPos.node;
     var hasStartParent = startNode.parentNode && startNode.parentElement;
 
-    if (browserUtils.isWebKit || !hasStartParent || !domUtils.isElementContainsNode(element, startNode)) {
+    var browserShouldUpdateSelection = browserUtils.isChrome && browserUtils.version < 58 || browserUtils.isSafari;
```
Comment (AlexanderMoskovkin) [inline src/client/automation/playback/type/type-text.js]: `browserShouldUpdateSelection` - from the var's name it seems that the browser should update selection itself and we shouldn't do it. But I suppose it should be `browserRequiresSelectionUpdating`. Am I right? /cc @georgiy-abbasov

Comment (georgiy-abbasov) [inline src/client/automation/playback/type/type-text.js]: Yes, you're right.

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

**Comment by AlexanderMoskovkin** [comment]
I think it would be good to run client tests on the chrome `beta` version to be notified about possible problems early. So, @georgiy-abbasov update the config to use `beta` version on Windows and `latest` version on MacOS (as it is now)

**Comment by AlexanderMoskovkin** [comment]
@testcafe-build-bot \retest

**Comment by georgiy-abbasov** [comment]
>> update the config to use beta version on Windows and latest version on MacOS (as it is now)
Ok

**Pull request merged**